templates/look-and-feel/components/fields.njk

```diff
@@ -85,6 +88,12 @@
            value="{{ option_value }}">
     <label for="{{ field.id }}-{{ option_value }}">
       {{ option.label }}
+      {% if option_description %}
+          <br>
```

Comment:
css should be used for layout not line breaks

Reply:
Sure, I've added a class instead so that downstream projects can control the styling of the span (default behaviour is just to show it inline). Any thoughts?

Reply:
looks fine, I've never used this project before but I've approved.